libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -105,7 +105,7 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
 #if CONFIG_HAL_BOARD == HAL_BOARD_PX4
     // @Param: SAFETY_MASK
     // @DisplayName: Channels to which ignore the safety switch state
-    // @Description: A bitmask which controls what channels can move while the safety witch has not been pressed
+    // @Description: A bitmask which controls what channels can move while the safety switch has not been pressed - 0:Ch1,1:Ch2,2:Ch3,3:Ch4,4:Ch5,5:Ch6,6:Ch7,7:Ch8
```

Comment:
Other params have it, that's why I added. Also it becomes quite hard to setup something if you can't tell how to set it up, without having either the source code available or best case scenario the wiki.